wordpress-mobile/WordPress-Android
# Display list of domains for a site on domains dashboard

Fetch and display list of domains for a site on domains dashboard

Fixes #15358 
Depends on this [FluxC PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2130)

<img width=320 src="https://user-images.githubusercontent.com/990349/135042985-f62eed83-ba6f-4e1f-90f2-79ad176b723e.png" />


To test:

- Enable AppSettings -> Debug settings -> SiteDomainsFeatureConfig ✅
- Return to My Site, Notice there's a new item under Configuration -> 🌐 Domains
- Click on Domains, takes your Site Domains Screen
- Notice that it display a list of domains under Your Site Domains
- Test with a site that has more than one domain on the site to see the above screen

NOTE:  Look and feel is not final, will be updated later.


## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
